tests/wicked/aggregate/sut/t16_teaming_lacp.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, LACP
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+use network_utils 'ifc_is_up';
+
+sub run {
+    my ($self, $ctx) = @_;
+    record_info('INFO', 'Teaming, LACP');
+    $self->setup_team('lacp', $ctx->iface(), $ctx->iface2());
+    validate_script_output('wicked ifstatus --verbose all|grep ' . $ctx->iface(),  sub { /enslaved/ });
+    validate_script_output('wicked ifstatus --verbose all|grep ' . $ctx->iface2(), sub { /enslaved/ });
+    if (!ifc_is_up('team0')) {
```

Comment:
we need to add some reaction in case interface would manage to start . does it ever happen ? don't know all details so might be wrong but my gut feel advice to add even a failure in case interface will start . Just to make sure that we don't miss this "miracle" :)

Reply:
ok, I will "die" if the "miracle" happens :)